datafusion/core/src/catalog/listing_schema.rs

```diff
@@ -59,18 +61,24 @@ impl ListingSchemaProvider {
     /// `path`: The root path that contains subfolders which represent tables
     /// `factory`: The `TableProviderFactory` to use to instantiate tables for each subfolder
     /// `store`: The `ObjectStore` containing the table data
+    /// `format`: The `FileFormat` of the tables
```

Comment:
I am not sure it matters, but this says `FileFormat` but the actual argument is a String